content/blog/2024-10-20-skywalking-go-toolkit-metrics-log/index.md

```diff
@@ -0,0 +1,159 @@
+
+SkyWalking Go currently provides the following three capabilities for users to manually report relevant information:
+* Trace
+* Metrics
+* Log
+
+This document aims to guide users on how to manually report log entries and metrics using the toolkit. For information on how to report trace linkage information with the toolkit, please refer to the [SkyWalking Go Toolkit Trace Detailed Explanation](https://skywalking.apache.org/blog/2023-10-18-skywalking-toolkit-trace/).
+
+Before diving deeper, you may want to check the [SkyWalking Go Agent Quick Start Guide](https://skywalking.apache.org/blog/2023-06-01-quick-start-with-skywalking-go-agent/) to learn how to use the SkyWalking Go Agent.
+
+The following sections will introduce how to use these interfaces in specific scenarios.
+
+## Import Trace Toolkit
+
+First, execute the following command in the root directory of the project:
+
+```shell
+go get github.com/apache/skywalking-go/toolkit
+```
+
+## Manually report logs
+
+In traceability, logs play a crucial role. They record detailed information about each request in the system, including timestamps, processing nodes, error messages, etc., which helps developers and operations teams quickly identify performance bottlenecks and the root causes of failures. By comparing logs from different requests, teams can analyze the flow of requests, optimize system architecture, and improve service response speed and stability.
+
+In the SkyWalking Go toolkit, manually reported logs will be **attached to the current context Span**, allowing us to associate specific log information with particular spans.
+
+First, we need to import the toolkit log package:
+
+```go
+"github.com/apache/skywalking-go/toolkit/logging"
+```
+
+We can build a simple web service that determines the validity of a username based on the request parameters. When the userName parameter is invalid, we log an error using the logging.Error API. This log will be attached to the currently active Span in the context.
+
+When recording logs, we can also append keyValues to the log information using variadic parameters, making the log entries more descriptive.
+
+For detailed usage documentation, please refer to [SkyWalking Go toolkit-logging](https://skywalking.apache.org/docs/skywalking-go/next/en/advanced-features/manual-apis/toolkit-log/).
+
+
+```go
+package main
+
+import (
+	"log"
+	"net/http"
+
+	_ "github.com/apache/skywalking-go"
+	"github.com/apache/skywalking-go/toolkit/logging"
+)
+
+func main() {
+	http.HandleFunc("/user", func(w http.ResponseWriter, r *http.Request) {
+		userName := r.URL.Query().Get("userName")
+		if len(userName) == 0 || userName != "root" {
+			// Log an entry, which will be attached to the currently active Span in the context.
+			// We can append log tags using variadic parameters.
+			logging.Error("deny user login", "userName", userName)
```

Comment:
Does the screenshot in the EN version still have Chinese? Could you help to replace it as english version?